lib/platform/bitbucket/README.md

```diff
@@ -0,0 +1,49 @@
+# BitBucket platform
+
+## Credentials setup
+
+### How to get
+
+BB_TOKEN for now is base64 encoded string of your `username:bbaAppPassword`.
```

Comment:
This section can be updated now

Reply:
why? it is internal BB adapter documentation

Reply:
now that you have implemented username/password, is there ever any need for users to manually encode anything?

Reply:
no, its not for renovates' users its for future BB adapter developers.